src/twisted/newsfragments/12398.feature

```diff
@@ -0,0 +1 @@
+twisted.internet.defer.Deferred processing is sligtly faster.
```

Comment:
I think that we should also mention the change to `Failure`

```suggestion
twisted.internet.defer.Deferred and twisted.python.failure.Failure processing are sligtly faster. This was implemented by using a the private twisted.internet.defer.Deferred._twistedPrivateIsDeferred attribute to detect a Deferred instance and twisted.python.failure.Failure._twistedPrivateIsFailure to detect a Failure instance. 
```

Reply:
Applied.